### DISCUSSION TITLE
Is there any support for $each currently?

### DISCUSSION THREAD

**Discussion by ScottLeWarne** [OData/AspNetCoreOData#1248] [General]
Hi, been using the library for awhile now and we are looking to implement the OData $each route.  Before starting out to implement ourselves I wanted to see if there was any support in the library for it at all that I am just missing?  If not, is anyone aware of any successful implementations out there?  And are there any plans to introduce support for it?  

Am thinking to follow the pattern for the $batch implementation as a starting point.

**Comment by julealgon** [comment]
It is not yet supported.

- https://github.com/OData/AspNetCoreOData/issues/1108#issuecomment-1830366764
> ...We currently don't support $each in ASP.NET Core OData library

by gathogojr

> ...$filter segment and $each are enabled in the ODL library, but they are not supported in the current ASP.NET Core OData.

by xuzhg

Check that issue for a bit more info.
